exp/runtime/internal/controllers/extensionconfig_controller.go

```diff
@@ -224,12 +228,16 @@ func discoverExtensionConfig(ctx context.Context, runtimeClient runtimeclient.Cl
 // Note: This was implemented to behave similar to the cert-manager cainjector.
 // We couldn't use the cert-manager cainjector because it doesn't work with CustomResources.
 func reconcileCABundle(ctx context.Context, client client.Client, config *runtimev1.ExtensionConfig) error {
+	log := ctrl.LoggerFrom(ctx)
+
 	secretNameRaw, ok := config.Annotations[runtimev1.InjectCAFromSecretAnnotation]
 	if !ok {
 		return nil
 	}
 	secretName := splitNamespacedName(secretNameRaw)
 
+	log.Info(fmt.Sprintf("Injecting CA Bundle into ExtensionConfig from secret %q", secretNameRaw))
```

Comment:
Would it be better to add the secretNameRaw as a k/v pair instead of using Sprintf?

Reply:
In my opinion it's not really one of the k/v pairs we would filter on / correlate logs with each other

Reply:
IMO k/v pairs aren't just for correlating, they're also to pull out specific values from structured logs, but I don't have a strong feeling either way on this one :slightly_smiling_face:

Reply:
Yup that's why I wrote `filter on` :). But I'm fine if we want to add it

Reply:
Leaving it as is for now.